server/src/controllers/Account.ts

```diff
@@ -54,6 +51,7 @@ export class AccountController {
   @OpenAPI({
     summary: 'Update the fields of an Account associated with id',
   })
+  @ResponseSchema(Account)
```

Comment:
For patch requests, it also takes in all of the attributes as optional parameters

Reply:
I think it works now;.